AMWin-RichPresence/AppleMusicScrobbler.cs

```diff
@@ -22,6 +22,10 @@ internal class AppleMusicScrobbler
         private string? lastSongID;
         private bool hasScrobbled;
 
+        //for personal use not sure if better way of doing this
+        string[] exemptArtists = {"Tyler, The Creator", "nothing,nowhere."};
```

Comment:
This doesn't look particularly nice to me. What's the purpose of these constants?

Reply:
Without having a array of exempt artists artists containing , or & will be split. It's an issue yeah that's why I'm wondering if it's possible to access the album artist or a list of artists on a song rather than parsing it from the currently playing song field.